<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>365</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>IRANYekan</family>
    <bold>false</bold>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>OZ Calc</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
qproperty-alignment: 'AlignVCenter | AlignRight';
border: 1px sold gray; 
padding: 16px;
}


background-color: white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(215,215,215);
font-size: 16px;
margin:1;
border: 2px sold gray;
}

QPushButton:pressed {
background-color: qlineargradient(x1: 0,y1: 0, x2: 0,y2: 1, stop: 0 #bebebe, stop: 1 #d7d7d7);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plusminus_btn">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(215,215,215);
font-size: 16px;
margin:1;
border: 2px sold gray;
}

QPushButton:pressed {
background-color: qlineargradient(x1: 0,y1: 0, x2: 0,y2: 1, stop: 0 #bebebe, stop: 1 #d7d7d7);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="percentage_btn">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(215,215,215);
font-size: 16px;
margin:1;
border: 2px sold gray;
}

QPushButton:pressed {
background-color: qlineargradient(x1: 0,y1: 0, x2: 0,y2: 1, stop: 0 #bebebe, stop: 1 #d7d7d7);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="division_btn">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(255,151,57);
font-weight: bold;
font-size: 16px;
color: #efefef;
margin: 1px;
border: 1px sold gray;
}

QPushButton:pressed {
background-color: qlineargradient(x1: 0,y1:0,x2:0,y2: 1, stop: 0 #ff7832, stop: 1 #ff9739);
}</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num7_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>121</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px sold gray;
background: #f6f7fa;
margin: 1px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0,y1: 0, x2: 0,y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiply_btn">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>121</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(255,151,57);
font-weight: bold;
font-size: 16px;
color: #efefef;
margin: 1px;
border: 1px sold gray;
}

QPushButton:pressed {
background-color: qlineargradient(x1: 0,y1:0,x2:0,y2: 1, stop: 0 #ff7832, stop: 1 #ff9739);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num8_btn">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>121</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px sold gray;
background: #f6f7fa;
margin: 1px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0,y1: 0, x2: 0,y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num9_btn">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>121</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px sold gray;
background: #f6f7fa;
margin: 1px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0,y1: 0, x2: 0,y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num6_btn">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>182</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px sold gray;
background: #f6f7fa;
margin: 1px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0,y1: 0, x2: 0,y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num5_btn">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>182</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px sold gray;
background: #f6f7fa;
margin: 1px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0,y1: 0, x2: 0,y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus_btn">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>182</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(255,151,57);
font-weight: bold;
font-size: 16px;
color: #efefef;
margin: 1px;
border: 1px sold gray;
}

QPushButton:pressed {
background-color: qlineargradient(x1: 0,y1:0,x2:0,y2: 1, stop: 0 #ff7832, stop: 1 #ff9739);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num4_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>182</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px sold gray;
background: #f6f7fa;
margin: 1px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0,y1: 0, x2: 0,y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num3_btn">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>243</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px sold gray;
background: #f6f7fa;
margin: 1px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0,y1: 0, x2: 0,y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num1_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>243</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px sold gray;
background: #f6f7fa;
margin: 1px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0,y1: 0, x2: 0,y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_btn">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>243</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(255,151,57);
font-weight: bold;
font-size: 16px;
color: #efefef;
margin: 1px;
border: 1px sold gray;
}

QPushButton:pressed {
background-color: qlineargradient(x1: 0,y1:0,x2:0,y2: 1, stop: 0 #ff7832, stop: 1 #ff9739);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num2_btn">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>243</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px sold gray;
background: #f6f7fa;
margin: 1px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0,y1: 0, x2: 0,y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num0_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>304</y>
      <width>122</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px sold gray;
background: #f6f7fa;
margin: 1px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0,y1: 0, x2: 0,y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equality_btn">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>304</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(255,151,57);
font-weight: bold;
font-size: 16px;
color: #efefef;
margin: 1px;
border: 1px sold gray;
}

QPushButton:pressed {
background-color: qlineargradient(x1: 0,y1:0,x2:0,y2: 1, stop: 0 #ff7832, stop: 1 #ff9739);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decimal_btn">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>304</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(215,215,215);
font-size: 16px;
margin:1;
border: 2px sold gray;
}

QPushButton:pressed {
background-color: qlineargradient(x1: 0,y1: 0, x2: 0,y2: 1, stop: 0 #bebebe, stop: 1 #d7d7d7);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
